### DISCUSSION TITLE
CONNECTION_FAILED

### DISCUSSION THREAD

**Discussion by Manfred-exe** [fredemmott/StreamDeck-Discord#65] [Q&A]
PLS Help nothing works after logging new in i tried everything i doesent even get a auth screen (Reinstalling the plugin,discord,stream deck. and i also restarted my computer a couple of times. i tried to run it with admin and without. i created 10 different aplications i changed the token I DID EVERYTHING. i stopped discord and stream deck and started it.........) but nothing worked can someone pls help me ?

**Comment by fredemmott** [comment]
- have you made sure you've entered the secret key and app id from the oauth page, not the main page?
- have you followed the instructions in TROUBLESHOOTING.md?
- hav eyou upgraded discord recently? A Discord update broke the authentication several weeks ago, but another update fixed it again since
